lib/rules/require-capitalized-constructors.js

```diff
@@ -11,6 +11,9 @@
  *
  * ```js
  * "requireCapitalizedConstructors": true
+ * "requireCapitalizedConstructors": [
```

Comment:
+1 for `allExpect`.

It's not clear to declare exceptions immediately after rule. That will look like "...require capitalized constructors. which ones? these... A B C" — definitely the opposite sense.

Good job at all. Thanks for contributing!